src/bindings/js/node/include/type_validation.hpp

```diff
@@ -99,6 +99,8 @@ bool validate(const Napi::CallbackInfo& info, std::vector<std::string>& allowed_
     return InputParameters<Ts...>::validate(info, std::index_sequence_for<Ts...>{});
 };
 
-std::string get_parameters_error_msg(const Napi::CallbackInfo& info, std::vector<std::string>& allowed_signatures);
+std::string get_parameters_error_msg(const Napi::CallbackInfo& info,
```

Comment:
Just to consider if this function is not to specialized instead build string for signatures it do more. 
What if name is not required or there will be need something else.

Maybe leave this function unchanged to build signature string and use assert macro to build final message where required.

```cpp
OPENVINO_THROW("Method 'my_method' called with invalid parameters\n", ov::js::get_parameters_signature_str(info, allowed_signatures));
```
Macro build message with exact location (file, line), so maybe give method name is not required at all.

Reply:
The exact location points to C++ implementation of method that may have a different name in JS API. 
Here it's  `readModelSync` vs `read_model_sync`, but in some cases it might be quite confusing for a user.